CHANGES.rst

```diff
@@ -180,18 +180,41 @@ Features
 
 
 
+Removals and backward incompatible breaking changes
+---------------------------------------------------
+
+- The shutdown logic in 3.9 waited on all tasks, which caused issues with some libraries.
+  In 3.10 we've changed this logic to only wait on request handlers. This means that it's
+  important for developers to correctly handle the lifecycle of background tasks using a
+  library such as ``aiojobs``. If an application is using ``handler_cancellation=True`` then
+  it is also a good idea to ensure that any :func:`asyncio.shield` calls are replaced with
+  :func:`aiojobs.aiohttp.shield`.
+
+  Please read the updated documentation on these points: \
```

Comment:
@bdraco here's the preview: https://aiohttp--8552.org.readthedocs.build/en/8552/changes.html#removals-and-backward-incompatible-breaking-changes. This didn't change the render. The right fix would be to have a list with leading dashes or asterisks.

Reply:
I'm missing something as the render looks fine to me
<img width="474" alt="Screenshot 2024-07-31 at 7 48 36 PM" src="https://github.com/user-attachments/assets/f0745add-4de3-4463-918b-57d936c2c92b">

Maybe its my screen size?

Reply:
Yes, it's your screen. Inspect with DevTools and you'll see that it's a paragraph with no line breaks.
![aiohttp-rm-changelog](https://github.com/user-attachments/assets/ba5faeeb-d7ee-47bf-b0a6-ce76e681618b)

FWIW, `\` in RST has a rather weird meaning. Sometimes, I use it to escape invisible spaces in between bits of syntax.

P.S. I'd rather have this generate `<li>`s.

Reply:
(that's at 200% zoom in Firefox)

Reply:
I did try leading dashes but that didn't work either as its a list inside of a list

Reply:
It would work, but might need an extra empty line after `:`. Feel free to ask me for RST advice next time — I have some experience with it and Sphinx internals+extensions.

Reply:
Here's an example of a nested list that does work and is still a part of said changelog document: https://github.com/aio-libs/aiohttp/pull/8066/files#diff-bd97a3018f85ae60eb7b42978db8f4ae548d3090cc95bc3e8d64f1d825d4c5ee

Reply:
Thanks.  That helps.